hostinit/setup.go

```diff
@@ -523,6 +516,13 @@ func (init *HostInit) ProvisionHost(ctx context.Context, h *host.Host) error {
 		if err := hostutil.RunSSHCommand("setup", hostutil.SetupCommand(h), sshOptions, *h); err != nil {
 			return errors.Wrap(err, "error running setup script on remote host")
 		}
+
+		if h.ProvisionOptions.OwnerId != "" &&
+			len(h.ProvisionOptions.TaskId) > 0 {
+			grip.Infof("Fetching data for task %s onto host %s", h.ProvisionOptions.TaskId, h.Id)
+			err = init.fetchRemoteTaskData(ctx, h.ProvisionOptions.TaskId, lcr.BinaryPath, lcr.ConfigPath, h)
+			grip.ErrorWhenf(err != nil, "Failed to fetch data onto host %s: %v", h.Id, err)
+		}
```

Comment:
I can understand moving this down a bit, but I might be inclined to keep it *above* line 513

Reply:
Does the setup script create the directory into which the artifacts are downloaded though?

Reply:
Yes, the setup scripts creates the directory.